platform_umbrella/apps/verify/test/home_base_test.exs

```diff
@@ -53,16 +54,57 @@ defmodule Verify.HomeBaseTest do
       end)
       |> Jason.decode!()
 
-    {:ok, team_id: team["id"]}
+    session
+    |> create_home_base(team["id"])
+    |> Wallaby.end_session()
+
+    :ok
+  end
+
+  verify "can create install", %{session: session, kind_install_worker: pid} do
+    install_name = "int-test-#{:rand.uniform(10_000)}"
+
+    {url, session} = navigate_to_home_base(session)
+
+    session =
+      session
+      |> home_base_login()
+      |> visit_relative("/installations/new")
+      |> assert_text("Create a new installation")
+      |> fill_in_name("installation[slug]", install_name)
+      |> find(Query.select("installation[kube_provider]"), &click(&1, Query.option("Kind")))
+      |> find(Query.select("installation[default_size]"), &click(&1, Query.option("Tiny")))
+      |> click(Query.button("Create Installation"))
+      # make sure we're on the success page
+      |> assert_text("Installation Created")
+      |> assert_path(~r|/installations/[\d\w-]+/success$|)
+
+    cmd = text(session, Query.css("pre"))
+
+    uri = URI.new!(url)
+
+    {:ok} = KindInstallWorker.start_from_command(pid, cmd, install_name, uri.host)
+  end
+
+  defp navigate_to_home_base(session) do
+    session =
+      session
+      |> visit("/traditional_services")
+      |> assert_has(table_row(text: "home-base-"))
+
+    id = text(session, Query.css("table tr td:first-child"))
+    link = Query.link("running_service_#{id}")
+
+    {attr(session, link, "href"), session |> click(link) |> last_tab()}
```

Comment:
We need the home-base URL so we can pass the host as allowed when we call `bi start`